client/src/features/checkout/components/PaymentSuccess/PaymentSuccess.js

```diff
@@ -29,9 +29,9 @@ const PaymentSuccess = ({ activeStep, steps }) => {
       </DialogContent>
 
       <DialogActions>
-        <Button onClick={handleCloseBttn} autoFocus>
+        <Button sx={{ m: '0 10px 10px 0' }} onClick={handleCloseBttn} autoFocus>
```

Comment:
please remove this. and dont interfere since we have a universal Alert Modal already in process with styling

Reply:
done